src/main/java/uk/gov/hmcts/darts/common/enums/TranscriptionWorkflowStageEnum.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.hmcts.darts.common.enums;
 
-public enum WorkflowStageEnum {
+public enum TranscriptionWorkflowStageEnum {
```

Comment:
Seems fine to have this based on v47 of the schema having a `CHARACTER VARYING` column for transcription_workflow.workflow_stage.

But I see v48 adds the confusion over the `trs_id` moving to the new table now.
Referring to @barry-jones50410 commit ae5a0e0ba010281691157a90c063a8776d1b9af6
--    remove transcription.trs_id and FK from transcription_status
--    remove transcription-workflow.workflow_stage, and replace with transcription_status.trs_id
--    add FK from transcription_status to transcription_workflow

I did like @barry-jones50410 suggestion to have the `trs_id` added to the transcription_comment table tbh if we decide there isn't a need to have the new `transcription_workflow` table as that would make it more obvious regarding the comments.

Reply:
The changes to teh data model are going to be implemented in a new ticket https://tools.hmcts.net/jira/browse/DMP-958